pycoast/cw_base.py

```diff
@@ -253,7 +253,7 @@ def _add_grid(self, image, area_def,
 
         # lons along major lat lines (extended slightly to avoid missing the
         # end)
-        lin_lons = np.arange(lon_min, lon_max + Dlon / 5.0, Dlon / 10.0)
+        lin_lons = np.arange(lon_min, lon_max + Dlon / 5.0, Dlon / 100.0)
```

Comment:
Doesn't this mean that there will be a lot more lines in the final output image?

Could `linspace` be used instead to make sure that the last value is included? https://docs.scipy.org/doc/numpy/reference/generated/numpy.linspace.html